pybamm/models/base_model.py

```diff
@@ -274,37 +275,60 @@ def timescale(self, value):
     def parameters(self):
         """Returns all the parameters in the model"""
         if self._parameters is None:
-            self._parameters = self._find_parameters()
+            self._parameters = self._find_symbols(
+                (pybamm.Parameter, pybamm.InputParameter)
```

Comment:
should this find `pybamm.FunctionParameter`s too?

Reply:
Oh yes, it should, that will make `print_parameter_info` and `parameter`  consistent with each other